testing/examples/run_instruction_apt_pkgs/cloudbuild.yaml

```diff
@@ -19,14 +19,32 @@ options:
     machineType: "N1_HIGHCPU_32"
 
 steps:
-  # Test the run_instruction_apt_pkgs example.
+  # Step to copy the rules_docker repo to a separete persistent volume.
+  # This is needed because the rules_docker repo and Bazel's output_base
```

Comment:
Add a sentence saying Source code is copied to /workspace directory by default on GCB.

Reply:
Done